develop/toolkit/parachains/e2e-testing/moonwall.md

```diff
@@ -0,0 +1,175 @@
+
+## Install Moonwall
+
+Moonwall can be installed either globally for system-wide access or locally within specific projects. This section covers both installation methods.
+
+!!! note
+    This documentation corresponds to Moonwall version `{{ dependencies.moonwall.version }}`. Ensure you're using the matching version to avoid compatibility issues with the documented features.
+
+### Global Installation
+
+Global installation provides system-wide access to the Moonwall CLI, making it ideal for developers working across multiple blockchain projects. Install it by running one of the following commands:
+
+=== "npm"
+
+    ```bash
+    npm install -g @moonwall/cli@{{ dependencies.moonwall.version }}
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm -g install @moonwall/cli@{{ dependencies.moonwall.version }}
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn global add @moonwall/cli@{{ dependencies.moonwall.version }}
+    ```
+
+Now, you should be able to run the `moonwall` command from your terminal.
+
+### Local Installation
+
+For better dependency management and version control within a specific project, local installation is recommended. First, initialize your project:
+
+```bash
+mkdir my-moonwall-project
+cd my-moonwall-project
+npm init -y
+```
+
+Then, install it as a local dependency:
+
+=== "npm"
+
+    ```bash
+    npm install @moonwall/cli@{{ dependencies.moonwall.version }}
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm install @moonwall/cli@{{ dependencies.moonwall.version }}
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add @moonwall/cli@{{ dependencies.moonwall.version }}
+    ```
+
+## Initialize Moonwall
+
+The `moonwall init` command launches an interactive wizard to create your configuration file:
+
+```bash
+moonwall init
+```
+
+During setup, you'll be prompted for the following parameters:
+
+- **`label`** - identifies your test configuration
+- **`global timeout`** - maximum time (ms) for test execution
+- **`environment name`** - name for your testing environment
+- **`network foundation`** - type of blockchain environment to use
+- **`tests directory`** - location of your test files
+
+Simply press `Enter` to accept defaults, or input custom values. You should see something like this:
+
+--8<-- 'code/develop/toolkit/parachains/e2e-testing/moonwall/init.html'
+
+The wizard generates a `moonwall.config` file:
+
+```json
+--8<-- 'code/develop/toolkit/parachains/e2e-testing/moonwall/init-moonwall.config.json'
+```
+
+The default configuration needs to be enhanced with specific details about your blockchain node and test requirements:
+
+- The `foundation` object defines how your test blockchain node will be launched and managed. For local development, the `dev` foundation is used which runs a local node binary
+    
+    !!!note
+        For more information about available options, check the [Foundations](https://moonsong-labs.github.io/moonwall/guide/intro/foundations.html){target=\_blank} section.
```

Comment:
```suggestion
For more information about available options, check the [Foundations](https://moonsong-labs.github.io/moonwall/guide/intro/foundations.html){target=\_blank} section.
```
This will be part of the new style guide updates, just fixing here so I don't have to update it later